pkg/master/import_known_versions_test.go

```diff
@@ -71,18 +71,20 @@ func TestTypeTags(t *testing.T) {
 // but are also registered in internal versions (or referenced from internal types),
 // so we explicitly allow tags for them
 var typesAllowedTags = map[reflect.Type]bool{
-	reflect.TypeOf(intstr.IntOrString{}):    true,
-	reflect.TypeOf(metav1.Time{}):           true,
-	reflect.TypeOf(metav1.Duration{}):       true,
-	reflect.TypeOf(metav1.TypeMeta{}):       true,
-	reflect.TypeOf(metav1.ListMeta{}):       true,
-	reflect.TypeOf(metav1.ObjectMeta{}):     true,
-	reflect.TypeOf(metav1.OwnerReference{}): true,
-	reflect.TypeOf(metav1.LabelSelector{}):  true,
-	reflect.TypeOf(metav1.GetOptions{}):     true,
-	reflect.TypeOf(metav1.ExportOptions{}):  true,
-	reflect.TypeOf(metav1.ListOptions{}):    true,
-	reflect.TypeOf(metav1.DeleteOptions{}):  true,
+	reflect.TypeOf(intstr.IntOrString{}):          true,
+	reflect.TypeOf(metav1.Time{}):                 true,
+	reflect.TypeOf(metav1.Duration{}):             true,
+	reflect.TypeOf(metav1.TypeMeta{}):             true,
+	reflect.TypeOf(metav1.ListMeta{}):             true,
+	reflect.TypeOf(metav1.ObjectMeta{}):           true,
+	reflect.TypeOf(metav1.OwnerReference{}):       true,
+	reflect.TypeOf(metav1.LabelSelector{}):        true,
+	reflect.TypeOf(metav1.GetOptions{}):           true,
+	reflect.TypeOf(metav1.ExportOptions{}):        true,
+	reflect.TypeOf(metav1.ListOptions{}):          true,
+	reflect.TypeOf(metav1.DeleteOptions{}):        true,
+	reflect.TypeOf(metav1.GroupVersionKind{}):     true,
+	reflect.TypeOf(metav1.GroupVersionResource{}): true,
```

Comment:
Since `AdmittanceReviewSpec` has a struct field of `GroupVersionKind` and `GroupVersionResource`, it was required.  Now, I'm not 100% sure this is required anymore since the similar requirement for these types to have `json` tags is no longer required but at one point it was, likely when we were using `admission/install` and the API tests were firing.